### PR TITLE
Migrate current pa connectivity component to typescript

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: src/utils/searchAPI.ts [hunk 1]
```diff
@@ -23,6 +24,44 @@ class SearchAPI {
...
+   * @param {String} areaType area type id, f.e. "ea", "states"
+   * @param {Number | String} areaId area id to request, f.e. "CRQ", 24
+   *
+   * @return {Promise<Object>} Array of objects with data of the protected areas
+   */
+  static requestDPC(
+    areaType: string,
+    areaId: string | number,
+    paNumber: number
+  ) {
```
Comment (erikasv) [inline src/utils/searchAPI.ts:59]: Please add the return type (you imprted it but isn't being used xD)

File: src/utils/searchAPI.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import axios, { CancelToken } from "axios";
```
Comment (erikasv) [inline src/utils/searchAPI.ts:1]: Please remoe CancelToken, we'll add it when we need it

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/utils/searchAPI.ts (hunk 2 above)
Comment (cazapatamar) [reply src/utils/searchAPI.ts]: Done!

File: src/utils/searchAPI.ts (hunk 1 above)
Comment (cazapatamar) [reply src/utils/searchAPI.ts]: Fixed!

**Review by erikasv — APPROVED** [review]

**Pull request merged**